app/src/behaviors/behavior_sensor_rotate_var.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sensor_rotate_var
+
+#include <drivers/behavior.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+
+#include <zmk/event_manager.h>
+#include <zmk/events/keycode_state_changed.h>
+#include <zmk/behavior_queue.h>
+#include <zmk/keymap.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+struct behavior_sensor_rotate_var_config {
+    char *cw_behavior_dev;
+    char *ccw_behavior_dev;
```

Comment:
You might be able to share a lot of code between this and the other version if you use the previous file's config struct for both version, but just initialize the params to 0 in this version. Then inside the "binding triggered" function, everything before and after the `switch` block could be identical.

Reply:
Yeah, so moved the whole callback to a "common" code, and added an `bool override_params` field to the config struct to determine if the parameters get updated or not.